src/qute/commands/registerCommands.ts

```diff
@@ -69,6 +70,55 @@ function registerJavaDefinitionCommand(context: ExtensionContext) {
   }));
 }
 
+/**
+ * Toggle for enabling/disabling Qute validation.
+ *
+ * @param context
+ */
+function registerQuteValidationToggleCommand(context: ExtensionContext) {
+  // Toggle off - add file to exclusion
+  context.subscriptions.push(commands.registerCommand(QuteSettings.QUTE_VALIDATION_ENABLED + QuteSettings.TOGGLE_OFF, async (uri?: Uri) => {
```

Comment:
Remove the constant with toggle off and on in settings and create 2
 constants in QuteClientCommandConstants

Reply:
Yes the full name without concat